docs/orchestration/index.md

```diff
@@ -159,14 +126,14 @@ stack {
 }
 ```
 
-For both equivalent configurations, the order of execution will be:
+In both configurations, the execution sequence will be:
 
 * stack-a
 * stack-b
 
-The same order of execution can be defined using **before**:
+You can also use the **before** field to define the execution order. For instance:
```

Comment:
The word "same" is important here, as the same final execution order is achieved by using `before` instead of `after`.